runner/runner.go

```diff
@@ -7,6 +7,7 @@ import (
 	"log"
```

Comment:
Cool, that you've tested this with ssl already, looks neat using it with curl (:.